File: src/main/java/hdzi/editstarters/EditStarters.java
package hdzi.editstarters;

import hdzi.editstarters.initializr.StarterInfo;

import java.util.Collection;

/**
 * 整个项目的顶级功能
 */
public interface EditStarters {
    /**
     * 删除starters
     */
    void removeStarters(Collection<StarterInfo> dependencies);

    /**
     * 添加starters
     */
    void addStarters(Collection<StarterInfo> dependencies);
}

File: src/main/java/hdzi/editstarters/EditStarters.kt
package hdzi.editstarters

import hdzi.editstarters.bean.StarterInfo

/**
 * Created by taojinhou on 2019/1/16.
 */
interface EditStarters {
    fun removeDependencies(dependencies: Collection<StarterInfo>)
    fun addDependencies(dependencies: Collection<StarterInfo>)
}